Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -33,6 +33,6 @@
     'HCal'        : 'Configuration.Geometry.GeometrySimHCAL_cff,Configuration.Geometry.GeometryRecoHCAL_cff',
     'Extended2023sim'   : 'Extended2023sim,Extended2023simReco',
     'Extended2023tilted'   : 'Extended2023tilted,Extended2023tiltedReco',
-    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco'
-    
+    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco',
+    'Extended2023GReco'   : 'Extended2023GReco,Extended2023GRecoReco'
```

Comment:
@ebrondol and @boudoul - by introducing these labels and pointing them to existing scenario you'd avoid file multiplication:
```
'Extended2023LReco'   : 'Extended2023sim,Extended2023simReco',
'Extended2023GReco'   : 'Extended2023sim,Extended2023simReco',
```